bin/dustc

```diff
@@ -1,7 +1,7 @@
 #!/usr/bin/env node
 
 var cli = require('cli').enable('glob', 'version'),
-    dust = require('../lib/server.js'),
+    dust = require('../index.js'),
```

Comment:
http://blog.izs.me/post/44149270867/why-no-directories-lib-in-node-the-less-snarky

grep "deeply regrettable"

My opinion on this is not held so strongly that I won't de-.js it if you feel really strongly on it.